client/components/editor.js

```diff
@@ -0,0 +1,42 @@
+Template.editor.helpers({
+  docid: function(){
+    var doc = Documents.findOne();
+    if (doc){
+      return doc._id
+    } else {
+      return undefined
+    }
+  },
+  config: function(){
+    return function(editor){
+      // console.log(editor)
+      editor.on("change", function(cm_editor, info){
+        // console.log(cm_editor.getValue())
+        $("#viewer_iframe").contents().find("html").html(cm_editor.getValue())
+        Meteor.call("addEditingUser")
+      })
+    }
+  }
+});
+
+Template.editingUsers.helpers({
+  users: function() {
+    var doc, eusers, users;
+    doc = Documents.findOne()
+    if (!doc){
+      return
+    }
+    eusers = EditingUsers.findOne()
+    if (!eusers){
+      return
+    }
+
+    users = []
+    var i = 0;
+    for (var user_id in eusers.users) {
```

Comment:
Be careful with `for/in` looping over arrays.
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in
- http://stackoverflow.com/questions/500504/why-is-using-for-in-with-array-iteration-such-a-bad-idea